app/controllers/account/tasks_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Account::TasksController < Account::AccountController
+  after_action :send_email_when_task_complete, only: :complete
```

Comment:
do it without after_action

just call sending email directly in controller

code must be obvious and readable

Reply:
yeah, you can send email within an action when u use `deliver_later`, it'll trigger async job and wouldn't cause a performance spike